apps/healthlocker/web/templates/support/index.html.eex

```diff
@@ -1,203 +1,154 @@
-      class: "hl-aqua flex-wrap underline" %>.
-      Free emotional support for people living in Lambeth, Southwark and Lewisham.
-      The service is run by peer supporters who have their own experience of facing and overcoming crisis.
-      Mon to Fri 6pm – 12 midnight, and Sat &amp Sun 12 midday – 12 midnight</p>
-
-    <h4 class="mt-3rem">National support</h4>
-    <p><strong>The Samaritans</strong> <%= link "116 123", to: "tel:116123", class: "hl-aqua flex-wrap underline" %>
-      is a national service (and you don't have to be suicidal to call for support)</p>
-
-    <p><strong>Childline</strong>
-      <%= link "0800 1111", to: "tel:0800 1111",
-      class: "hl-aqua flex-wrap underline" %> or
-      <%= link "chat online to a counsellor",
-      to: "https://www.childline.org.uk/get-support/1-2-1-counsellor-chat/",
-      class: "hl-aqua flex-wrap underline" %>.
-      A free confidential line for children &amp young people (you don’t have to give your name if you don’t want to)</p>
-
-    <p><strong>FRANK</strong> <%= link "0300 123 6600", to: "tel:0300 123 6600",
-      class: "hl-aqua flex-wrap underline" %>
-      A free confidential helpline for anyone concerned about drug use.
-      <%= link "Chat online 2-6pm",
-      to: "http://www.talktofrank.com/livechat",
-      class: "hl-aqua flex-wrap underline" %> or read
-      <%= link "what to do", to: "http://www.talktofrank.com/emergency-help",
-      class: "hl-aqua flex-wrap underline" %>
-      if a friend or relative is having a bad reaction to a drug</p>
-
-    <p><strong>NHS</strong> <%= link_to("111", to: "tel:111") %> call for urgent, but non-emergency advice</p>
-
-    <p><strong>Emergency services</strong> <%= link_to("999", to: "tel:999") %>
-      or find your nearest <%= link_to("A&E", to: "http://www.nhs.uk/Service-Search/Accident-and-emergency-services/LocationSearch/428") %></p>
-  </div>
+<p>
+    If you have immediate concerns about your physical safety and/ or wellbeing, or the physical safety and
+      wellbeing of your child, you should contact the emergency services:
+</p>
+    <p><strong>Emergency services</strong> <%= link "999", to: "tel:999" %>
+      or find your nearest <%= link "A&E", to: "http://www.nhs.uk/Service-Search/Accident-and-emergency-services/LocationSearch/428" %></p>
+
+    <p><strong>NHS</strong> <%= link "111", to: "tel:111" %> call for urgent, but non-emergency advice.
+    </p>
+    <p>You can also contact your <strong>out of hours GP service.</strong> Check the website of your GP surgery to find out details of how to contact this service.
+    </p>
+</div>
 
   <p>
-    If you need help or support using Healthlocker, please email
-    <%= link_to "healthlocker@slam.nhs.uk", to: "mailto:healthlocker@slam.nhs.uk" %>
+    If you need help or support using Headscape Focus, please email
+    <%= link "oxl-tr.icsadhdservice@nhs.net", to: "mailto:oxl-tr.icsadhdservice@nhs.net", class: "hl-aqua flex-wrap underline" %>.
```

Comment:
@SimonLab the `flex-wrap` was included so this would happen. Otherwise there was strange spaces from a link moving down to the next line, and it was preferable to have the links wrap like this rather than break up the text.